packages/core/strapi/src/cli/utils/data-transfer.ts

```diff
@@ -210,14 +210,18 @@ const errorColors = {
 } as const;
 
 const formatDiagnostic = (
-  operation: string
+  operation: string,
+  info?: boolean
```

Comment:
Same comment as CLI command, should this accept an actual level rather than a boolean?

Reply:
I still think it should be a boolean, just `--verbose` rather than info